sdk/textanalytics/Azure.AI.TextAnalytics/src/HealthcareEntityRelationRole.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Determines the relation role for the entity.
+    /// </summary>
+    public partial class HealthcareEntityRelationRole
+    {
+        internal HealthcareEntityRelationRole(HealthcareEntityInternal entity, string entityName)
+        {
+            Name = entityName;
+            Entity = new HealthcareEntity(entity);
+        }
+
+        /// <summary>
+        /// Healthcare entity.
+        /// </summary>
+        public HealthcareEntity Entity { get; }
+
+        /// <summary>
+        /// Determines the name for the role. It be an Entity, Attribute etc.
```

Comment:
the values will change with the new model version, so maybe better if we don't put them as example here.
We should consider putting them in an extensible enum like Python did

Reply:
updated. enum change should come from swagger.